etc/org.opencastproject.speechtotext.impl.SpeechToTextServiceImpl.cfg

```diff
@@ -3,5 +3,5 @@
 #job.load.speechtotext=0.8
 
 # Select STT engine.
-# Available engines: vosk, whisper | Default: vosk
-SpeechToTextEngine.target=(enginetype=whisper)
+# Available engines: vosk, whisper, whispercpp | Default: vosk
+#SpeechToTextEngine.target=(enginetype=vosk)
```

Comment:
So, this config doesn't actually *do* anything, since `STTSImpl` does not parse it, or change its hardcoded (vosk) binding.

What this does do is cause `STTSImpl` to hang waiting for its bindings to finish :)